hangman_l.rb

```diff
@@ -0,0 +1,174 @@
+    while @guess.length > 1 || @guess.to_i != 0 || @guess == "0"
+
+      puts "Please enter a valid character."
+      @guess = gets.chomp.downcase
+
+    end
+
+  end
+
+# Generates ascii dragon art
+  def print_art
+    if @wrong_counter == 0
+      puts "
+                ._____.
+              .          .
+            :              :
+           ;                ;
+           |                |
+           |                |
+           ;                ;
+            :              /
+    ---------.           .---------
+                ;.____.;
+    ".colorize(:green)
+
+    elsif @wrong_counter == 1
+      puts "
+                ._____.
+              .          .
+            :              :
+           ;       ^        ;
+           |       |        |
+           |             ^  |
+           ;    |        |  ;
+            :  \\       /  /
+    ---------.  |      | .---------
+                ;.____.;
+    ".colorize(:light_black)
+
+    elsif @wrong_counter == 2
+      puts "
+                .__^__.
+              .           .
+            :    /         :
+           ;       ^    /   ;
+           |   *   |        |
+           |    *        ^  |
+           ;    |        |  ;
+            :  \\       /  /
+    ---------.  |      | .---------
+                ;.____.;
+    ".colorize(:white)
+
+    elsif @wrong_counter == 3
+      puts "
+                   (\\^~^/)
+                    )d b(
+                  \\(  _  )/
+                   ((q_p))
+                . _________ .
+              :    /         :
+             ;       ^    /   ;
+             |   *   |        |
+             |    *        ^  |
+             ;    |        |  ;
+              :  \\       /  /
+      ---------.  |      | .---------
+    ".colorize(:yellow)
+    elsif @wrong_counter == 4
+      puts "
+                 (\\^~^/)
+          ._____  )d b(  _____.
+         .__)_   \(  _  )/  _(__.
+         ._)_    ((q_p))    _(_.
+           \\__  .-)\\u/(-.  __/
+            \\ :\\,-.   .   ,-./
+             |||\\,       ,/||
+             | \\    /\\   /  |
+             ;    \\ -<      ;
+              :   \\ -. \\_  /
+      ---------. \\       .---------
+                 ;.____.;
+    ".colorize(:magenta)
+    elsif @wrong_counter == 5
+      puts "
+           (\\               /)
+          __)\\             /(__
+         __)_ \\  (\\!~!/)  / _(__
+        __)_ `.\\  )d b(  /.' _(__  *YOU'RE NOT MY MOMMY*
+      ___)_ `. \\\\(  _  )// .' _(___  *BREATHES FIRE*
+       )_  `. \\  ((q_p))  / .'  _(_    *GAME OVER*
+       _)`.  \\  ,-)\\_/(-.  /  .'(_
+        _) ,-._/v/vvvvv\\v\\_,-. (_
+        _)///(._/v(___)v\\_.)\\\\\\(_
+         \\_ ___/v(_____)v\\___ _/
+           /vvv\\(_______)/vvv\\
+           \\vvv/v(_____)v\\vvv/
+           _\\vv\\\\v(___)v//vv/_
+          '>_`  \\`-._.-'/  '_<`
+          ' >_,-'       `-._<`
+
+      ".colorize(:red)
+      exit
+
+    end
+  end
+
```

Comment:
I love this ASCII art!! 💜🐲

Also, for readability, you might consider using `case...when` instead of a series of `elsif` statements. So,
```
case @wrong_counter
when 1
   puts ...
when 2
  puts ...
```
etc.